src/EFCore.MySql/Extensions/MySqlComplexTypePropertyBuilderExtensions.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Pomelo Foundation. All rights reserved.
+// Licensed under the MIT. See LICENSE in the project root for license information.
+
+using JetBrains.Annotations;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Microsoft.EntityFrameworkCore.Utilities;
+
+// ReSharper disable once CheckNamespace
+namespace Microsoft.EntityFrameworkCore
+{
+    /// <summary>
+    ///     MySQL specific extension methods for <see cref="ComplexTypePropertyBuilder" />.
+    /// </summary>
+    public static class MySqlComplexTypePropertyBuilderExtensions
+    {
+        #region Computed
+
```

Comment:
Should be `CharSet` or could be removed (since there is only one kind of methods in here at the moment).